backend/main/src/User/Domain/User.ts

```diff
@@ -7,10 +7,13 @@ interface IUserProps {
 
 /**
  * This class is user class.
- * The name of user must be less than equal to 30.
+ * The name of user must be less than equal to 30a and greater than equal to 3.
```

Comment:
This comment should be as follows.
`The name of user must be less than equal to 30 and greater than equal to 1`

Reply:
Thank you. I missed it.